addon/components/sequential-render.js

```diff
@@ -276,9 +276,10 @@ export default Component.extend({
   updateRenderStates() {
     set(this, 'isFullFilled', true);
 
-    run.next(() => {
+    let runNext = run.next(() => {
       this.reportRenderState();
     });
+    get(this, 'renderStates').scheduledCalls.pushObject(runNext);
```

Comment:
Do you see any possibilities to remove from this queue when it is executed so that we don't have to cancel those redundantly?

Reply:
Thanks for this point, have addressed in latest commit